tests/api2/test_disk_wipe.py

```diff
@@ -24,6 +22,40 @@ def test_disk_wipe_exported_zpool_in_disk_get_unused():
         assert disk['exported_zpool'] is None
 
 
+def test_disk_wipe_partition_clean():
+    """
+    Confirm we clean up around the middle partitions
+    """
+    signal_msg = "ix private data"
+
+    disk = call("disk.get_unused")[0]["name"]
```

Comment:
Do we need to SKIP or FAIL the test if there are no unused disks available?

Reply:
We don't need to skip tests if some resources are unavailable. VM environment misconfiguration will leave those tests to be not executed silently (at least with our current Jenkins).